python/ct/crypto/in_memory_merkle_tree_test.py

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env python
```

Comment:
Ignore this - I think it'll be necessary for Travis to run the test successfully.